src/monorepo-shipit/config/docs-adeira-dev.js

```diff
@@ -1,7 +1,7 @@
 // @flow strict
 
 module.exports = {
-  getStaticConfig() {
+  getStaticConfig(): { repository: string, ... } {
```

Comment:
Why not `{| repository: string |}`?

Reply:
Changed. It's autogenerated code and I didn't touch it unless it's a total disaster. :)